libbeat/beat/beat.go

```diff
@@ -104,7 +104,14 @@ func Run(name string, version string, bt Beater) {
 
 	// Runs beat inside a go process
 	go func() {
-		b.Start()
+		err := b.Start()
+
+		if err != nil {
+			// TODO: detect if logging was already fully setup or not
```

Comment:
That would actually be nice. So we could use logp.* in all cases and it would decide if it needs fmt.* or not.